paper/paper.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'nyaml: A Format Converter for NeXus Data Model.'
+tags:
+    - Python
+    - NeXus
+    - NXDL
+authors:
+  - name: Adrian M. Price-Whelan
+    orcid: 0000-0000-0000-0000
+    equal-contrib: true
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+  - name: Author Without ORCID
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 2
+  - name: Author with no affiliation
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: 3
+  - given-names: Ludwig
+    dropping-particle: van
+    surname: Beethoven
+    affiliation: 3
+affiliations:
+ - name: Lyman Spitzer, Jr. Fellow, Princeton University, United States
+   index: 1
+   ror: 00hx57361
+ - name: Institution Name, Country
+   index: 2
+ - name: Independent Researcher, Country
+   index: 3
+date: 13 August 2017
+bibliography: paper.bib
+---
+# Summary
+# Statement of need
+# NYAML to NXDL
+# NXDL to NYAML
+# Evaluation from NAIC
```

Comment:
Evaluation from the NIAC

Reply:
We shall though not have these acronyms in the section header unless we are sure to have these introduced in preceeding sections, overall though spelling these acronyms out is better is my opinion

Reply:
Abbreviation of the acronyms is included.